spec/features/email_spec.rb

```diff
@@ -29,15 +29,16 @@
 
   scenario "logged in as Organizer after sending an Email to the applicants the event application status gets locked" do
     login(:organizer)
-    @event.application_status_locked = false
+    @event.acceptances_have_been_sent = false
```

Comment:
I think this test should work independent of the current state. After sending an email the participant selection should be locked. You are right, that a emails can only be send in selection phase, but I think that should not be part of the test.